### PR TITLE
Ingestor

### PR DESCRIPTION
**Pull request by lucamolteni**
First Draft of  https://github.com/kiegroup/akrivis/issues/6

### CODE REVIEW

**Comment by lucamolteni** [comment]
This is the PR for the React Frontend https://github.com/kiegroup/backstage-plugins/pull/3

**Review by Rikkola — APPROVED** [review]
I think this is fine as it is for now. License headers is really the only thing I am missing.

I added few other comments to open a discussion, we can do it here, in chat or in a separate meeting.

File: akrivis-ingestor/src/main/java/org/kie/akrivis/scheduler/JobRepository.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import io.quarkus.panache.common.Sort;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.persistence.NoResultException;
+import jakarta.persistence.NonUniqueResultException;
+
+import java.util.List;
+import java.util.Optional;
+
+@ApplicationScoped
+public class JobRepository implements PanacheRepository<Job> {
```
[the remark this answers is not in the document]
Comment (Rikkola) [reply akrivis-ingestor/src/main/java/org/kie/akrivis/scheduler/JobRepository.java:13]: Coming back after reading the doc. If the Ingestor gets all the data for running from the database, this makes more sense. Then there will be one less image in the pod to configure.

File: akrivis-ingestor/pom.xml [hunk 2]
```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.kie.akrivis</groupId>
+    <artifactId>akrivis-ingestor</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <properties>
+        <compiler-plugin.version>3.12.1</compiler-plugin.version>
+        <maven.compiler.release>21</maven.compiler.release>
```
[the remark this answers is not in the document]
Comment (lucamolteni) [reply akrivis-ingestor/pom.xml:10]: Fixed in https://github.com/kiegroup/akrivis/pull/8/commits/a38f0a8bc266aa334bc8a7f2e6e651c95e99e09b

File: akrivis-ingestor/src/main/java/org/kie/akrivis/scheduler/IngestorHttpClient.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package org.kie.akrivis.scheduler;
```
[the remark this answers is not in the document]
Comment (lucamolteni) [reply akrivis-ingestor/src/main/java/org/kie/akrivis/scheduler/IngestorHttpClient.java:1]: Discussed in chat, not required currently

**Pull request merged**